### PR TITLE
Update deployment options doc

### PR DESCRIPTION
**Pull request by Amruta-Ranade**
Addresses https://cockroachlabs.atlassian.net/browse/DOC-8861

### CODE REVIEW

**Review by mowinslow2 — APPROVED** [review]
May want to include something about [private clusters](https://deploy-preview-17909--cockroachdb-docs.netlify.app/docs/cockroachcloud/private-clusters) too in the Advanced section? But approved, thank you!

**Comment by Amruta-Ranade** [comment]
> private clusters

As per the Slack convo, we don't need to call out Private Clusters on this page. Let me know if you disagree. cc @jramos-crl

**Review by florence-crl — APPROVED** [review]
LGTM! one nit

**Review by jramos-crl — APPROVED** [review]
LGTM!

File: src/current/v22.2/choose-a-deployment-option.md [hunk 1]
```diff
@@ -61,17 +65,12 @@ Cockroach Labs offers four ways to deploy CockroachDB: three managed services&md
         <li>Applications that need to be PCI compliant.</li>
       </ul></td>
       <td><ul>
```
[the remark this answers is not in the document]
Comment (Amruta-Ranade) [reply src/current/v22.2/choose-a-deployment-option.md]: Good catch! Fixed

**Pull request merged**